crypto/ocsp/ocsp_test.cc

```diff
@@ -39,6 +39,12 @@ static const time_t invalid_after_ocsp_expire_time_sha256 = 1937505764;
 #define OCSP_URL_PARSE_SUCCESS 1
 #define OCSP_URL_PARSE_ERROR 0
 
+#define OCSP_NONCE_EQUAL 1
+#define OCSP_NONCE_BOTH_ABSENT 2
+#define OCSP_NONCE_RESPONSE_ONLY 3
+#define OCSP_NONCE_NOT_EQUAL 0
+#define OCSP_NONCE_REQUEST_ONLY -1
```

Comment:
not a blocker: But I'm wondering if the user experience would better if these were just public anyways? Then you can use them in the `OCSP_check_nonce` implementation as well.

Reply:
Good idea, I'd originally created these with only testing in mind, but these totally make sense in a broader context as well.